.gitignore

```diff
@@ -21,11 +21,11 @@ __pycache__
 /cmd/podman/varlink/ioprojectatomicpodman.go
 /cmd/podman/varlink/iopodman.go
 .gopathok
-test/e2e/e2e.coverprofile
```

Comment:
Why? I think this is useful to prevent it accidentally being committed

Reply:
Because we ignore all `*.coverprofile` now